microsoft-azure-sb/3.2/modules/ROOT/pages/azure-service-bus-connector-reference.adoc

```diff
@@ -243,6 +243,8 @@ This operation receives a message from a specific queue or subscription. In this
 
 This operation receives a maximum of maxMessageCount messages from a specific queue or subscription. In this version of the connector, only Immediate Receive Mode (`RECEIVEANDDELETE`) is supported.
 
+Due to a limitation on the SDK that Azure Service Bus Connector uses provided by Azure, in some cases, the *Receive Batch* operation can receive fewer messages than the ones that are available. To receive all the remaining messages, call the operation again.
```

Comment:
It is not clear what "provided by Azure" modifies - is it the SDK? 
If so, "Due to a limitation on the Azure-provided SDK..."

Reply:
all the > all of the